docs/recording/recording.md

```diff
@@ -0,0 +1,190 @@
+<prettier-ignore-end>
+
+## Starting LTTng session via ROS launch
+
+You can start LTTng session using ROS launch system. When you start a LTTng session in one terminal, you have to open another terminal for executing a target application as explained above. Operating multiple terminals is laborious for users. Launching LTTng session along with application by ROS launch is a reasonable way to apply CARET repeatedly.
+
+1. Create a launch file for a target application in ROS general manner if you haven't made it
+
+   ```py
+   # launch/end_to_end_sample.launch.py
+   import launch
+   import launch_ros.actions
+
+
+   def generate_launch_description():
+       return launch.LaunchDescription([
+           launch_ros.actions.Node(
+               package='caret_demos', executable='end_to_end_sample', output='screen'),
+       ])
+   ```
+
+2. Add description to start a LTTng session
+
+   ```py
+   # launch/end_to_end_sample_with_lttng_session.launch.py
+   import launch
+   import launch_ros.actions
+   from tracetools_launch.action import Trace
+
+
+   def generate_launch_description():
+       return launch.LaunchDescription([
+           Trace(
+               session_name='e2e_sample',
+               events_kernel=[],
+               events_ust=['ros2*']
+           ),
+           launch_ros.actions.Node(
+               package='caret_demos', executable='end_to_end_sample', output='screen'),
+       ])
+   ```
+
+3. Launch a target application and a LTTng session via the launch file
+
+   - Environment settings are still needed, but all operations are performed in just one terminal
+
+   ```sh
+   source /opt/ros/humble/setup.bash
+   source ~/ros2_caret_ws/install/local_setup.bash
+   source ~/ros2_ws/install/local_setup.bash
+
+   export LD_PRELOAD=$(readlink -f ~/ros2_caret_ws/install/caret_trace/lib/libcaret.so)
+
+   source ./caret_topic_filter.bash
+
+   ros2 launch caret_demos end_to_end_sample_with_lttng_session.launch.py
+   ```
+
+## Advanced: Useful settings for launch file
+
+- The following script shows advanced settings for a launch file
+- `caret_session` option is used to set a session name. If not assigned, datetime (YYYYMMDD-HHMMSS) is used
+- `caret_light` option is used to add another event filter. Only events specified in `caret_event` variable are recorded. In this exsample, high level events, such as ros2:rclcpp layer, are recorded if `caret_light` is set to "true". This setting will be helpful to record a huge application.
```

Comment:
typo

```suggestion
- `caret_light` option is used to add another event filter. Only events specified in `caret_event` variable are recorded. In this example, high level events, such as ros2:rclcpp layer, are recorded if `caret_light` is set to "true". This setting will be helpful to record a huge application.
```

Reply:
@hsgwa 
Thank you for correcting it. mm